scripts/mock-upcoming-ipodata.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
should we have mock data in the excel?

Reply:
any way we are going to remove that so easy to have it in json as others as well.